ironfish/src/rpc/routes/accounts/importAccount.ts

```diff
@@ -47,7 +47,10 @@ router.register<typeof ImportAccountRequestSchema, ImportAccountResponse>(
   ImportAccountRequestSchema,
   async (request, node): Promise<void> => {
     const account = await node.accounts.importAccount(request.data.account)
-    void node.accounts.startScanTransactionsFor(account)
+
+    if (request.data.rescan !== false) {
```

Comment:
We generally don't add equality checks to things that are already bools

```suggestion
    if (request.data.rescan) {
```